client/components/DateFormat.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+function DateFormat({ rawDate }) {
+  const dateToMils = Date.parse(rawDate);
+  const d = new Date(dateToMils);
+
+  const ye = new Intl.DateTimeFormat('en', { year: 'numeric' }).format(d);
+  const mo = new Intl.DateTimeFormat('en', { month: 'long' }).format(d);
+  const da = new Intl.DateTimeFormat('en', { day: '2-digit' }).format(d);
+
+  const newDate = `${mo} ${da}, ${ye}`;
```

Comment:
you could actually shorten this code here to look something like this.  
`const createdAt = new Intl.DateTimeFormat('en-US', {
    month: 'long',
    day: 'numeric',
    year: 'numeric',
  }).format(new Date(answer.created_at));`
and it will output like this:  December 14, 2019

otherwise that looks good besides the sheer number of variables.